nixos/modules/services/web-apps/miniflux.nix

```diff
@@ -116,7 +116,7 @@ in
         RestrictRealtime = true;
         RestrictSUIDSGID = true;
         SystemCallArchitectures = "native";
-        SystemCallFilter = [ "@system-service" "~@privileged" "~@resources" ];
+        SystemCallFilter = [ "@system-service" ];
```

Comment:
Is @privileged also needed? If so, for what?

```
❯ systemd-analyze syscall-filter @privileged
@privileged
    # All system calls which need super-user capabilities
    @chown
    @clock
    @module
    @raw-io
    @reboot
    @swap
    _sysctl
    acct
    bpf
    capset
    chroot
    fanotify_init
    fanotify_mark
    nfsservctl
    open_by_handle_at
    pivot_root
    quotactl
    setdomainname
    setfsuid
    setfsuid32
    setgroups
    setgroups32
    sethostname
    setresuid
    setresuid32
    setreuid
    setreuid32
    setuid
    setuid32
    vhangup

Reply:
> vm-test-run-miniflux> withoutSudo # [   27.072224] audit: type=1326 audit(1666527247.100:94): auid=4294967295 uid=64274 gid=64274 ses=4294967295 subj=kernel pid=905 comm="miniflux" exe="/nix/store/jwbva1d4xh3mrivhvry6mqvfn85nddam-miniflux-2.0.39/bin/miniflux" sig=31 arch=c000003e syscall=160 compat=0 ip=0x4043ee code=0x80000000

That's setrlimit, which is part of `@resources`. My theory is that the golang runtime started using that syscall in 1.19.

Please restore `"~@privileged"`, it is not required and should be forbidden.

Reply:
Ah, I think I understand where the confusion stems from.

Systemd in `systemd-analayze security` will complain if we don't deny `@privileged`. I do wonder why that is, because setting up an allow list alone should be sufficient.